src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/ProcessOutputLineListener.java

```diff
@@ -17,9 +17,13 @@
 package com.google.cloud.tools.app.impl.cloudsdk.internal.process;
 
 /**
- * Process listener that can be used with {@link DefaultProcessRunner} to read the process std or
- * error output line by line.
+ * Process listener that can be used with {@link DefaultProcessRunner} to read the merged stdout
```

Comment:
Ah I get it.  This is generic.  Will change docs.

BTW this looks like it's used as part of the public api, why is under impl....internal?